pages/common/aws-ecr.md

```diff
@@ -0,0 +1,36 @@
+# aws ecr
+
+> Push, pull, and manage container images.
+> More information: <https://awscli.amazonaws.com/v2/documentation/api/latest/reference/ecr/index.html>.
+
+- Authenticate Docker with the default registry (username is AWS):
+
+`aws ecr get-login-password --region {{region}} | {{docker login}} --username AWS --password-stdin {{aws_account_id}}.dkr.ecr.{{region}}.amazonaws.com`
```

Comment:
```suggestion
- Authenticate Docker with the default registry:

`aws ecr get-login-password --region {{region}} | {{docker login}} --username {{username}} --password-stdin {{aws_account_id}}.dkr.ecr.{{region}}.amazonaws.com`
```

Reply:
Correct me if I am mistaken. For general usage with ecr, the username is AWS.

Reply:
I note that username is AWS, because that is what AWS sets it to when you use ecr.

Reply:
I imagine most don't use the default username.

Also from what I have seen here, we only use examples for the variables when its something people might not understand otherwise, a username is a simple concept 

While (for example) something like a block device on Linux might not be understood

Reply:
Let's keep `AWS` here — it's not a username that a user would pick out, like an IAM user or anything.